gradlew

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
```

Comment:
oh well, new version of gradle uses `sh` for it's wrapper scripts not `bash`, that's nice xDDDD